TF-15131 Update product documentation with new default project behavior (#689)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/workspaces.mdx

| `data.attributes.working-directory`             | string  | (nothing) | A relative path that Terraform will execute within. This defaults to the root of your repository and is typically set to a subdirectory matching the environment when multiple environments exist within the same repository. |
| `data.attributes.setting-overwrites` | object  | none |   The keys in this object are attributes that have organization-level defaults. Each attribute key stores a boolean value which is `true` by default. To overwrite the default inherited value, set an attribute's value to `false`. For example, to set `execution-mode` as the organization default, set `setting-overwrites.execution-mode` to `false`. |
| `data.relationships` | object | none | Specifies a group of workspace associations. |

> | `data.relationships.project.data.id` | string  | default project | The ID of the project to create the workspace in. You must supply a project ID if no default project exists. You must have permission to create workspaces in the project, either by organization-level permissions or team admin access to a specific project. |

Reviewer: Small style changes for clarity:
```suggestion
| `data.relationships.project.data.id` | string  | default project | The ID of the project to create the workspace in. You must supply a project ID if you do not have a default project. You must have permission to create workspaces in the project, either by organization-level permissions or team access to that project. |
```
